templates/zerver/emails/new_login/new_login_alert.txt

```diff
@@ -0,0 +1,18 @@
+{# Informs user of a new login to their account #}
+
+Hello, {{ user.short_name | title }}.
+
+This is a notification that a new login to your Zulip account has just occured.
+
+Details
+Account: {{ user.email }}
+Date/Time: {{ device_info.login_time }}
```

Comment:
We should add `Server: {{ realm.uri }}` in here, to make clear which Zulip server they logged into.